BitmovinConvivaAnalytics/Classes/Helper/BitmovinPlayerHelper.swift

```diff
@@ -29,7 +29,7 @@ final class BitmovinPlayerHelper: NSObject {
         }
     }
 
-    var version: String {
+    static var version: String {
```

Comment:
There is no need to have that on the instance. Most likely this is legacy from the old `CFBundleVersion` extraction